File: Python/1035.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

def converte_lista_int(lista_str):
    lista_int = []
    for numero in lista_str:
        lista_int.append(int(numero))
    return lista_int


A, B, C, D = converte_lista_int(input().split())

if B > C and D > A and (C + D) > (A + B) and C > 0 and D > 0 and not(A % 2):
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: C++/1035.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    unsigned long A, B, C, D;
    int flag = 0;

    cin >> A >> B >> C >> D;

    if (B > C && D > A)
    {
        if (C+D > A+B)
        {
            if (C >= 0 && D >= 0)
            {
                if (A % 2 == 0)
                {
                    cout << "Valores aceitos" << endl;
                    flag = 1;
                }
            }
        }
    }
    if (!flag) cout << "Valores nao aceitos" << endl;
    return 0;
}